### PR TITLE
add dli queue service support

### PR DESCRIPTION
**Pull request by hmbig2**
add dli queue service support

### CODE REVIEW

File: openstack/dli/v1/queues/requests.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package queues
+
+import (
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/common/tags"
+)
+
+// CreateOpts contains the options for create a service. This object is passed to Create().
+type CreateOpts struct {
+	// Name of a newly created resource queue. The name can contain only digits, letters, and underscores (_), but cannot contain only digits or start with an underscore (_). Length range: 1 to 128 characters.
```
Comment (ShiChangkuo) [inline openstack/dli/v1/queues/requests.go:10]: no more than 120 characters in one line

File: openstack/dli/v1/queues/requests.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	QueueName string `json:"queue_name" required:"true"`
+
+	// Indicates the queue type. The options are as follows:
+	// sql
+	// general
+	// all
+	// NOTE:
+	// If the type is not specified, the default value sql is used.
+	QueueType string `json:"queue_type"`
```
Comment (ShiChangkuo) [inline openstack/dli/v1/queues/requests.go:19]: suggest to add `omitempty` for optional fields

File: openstack/dli/v1/queues/requests.go [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+	return
+}
+
+func Delete(c *golangsdk.ServiceClient, queueName string) (r DeleteResult) {
+	reqOpt := &golangsdk.RequestOpts{OkCodes: []int{200}}
+	_, r.Err = c.Delete(resourceURL(c, queueName), reqOpt)
+	return
+}
+
+func Get(c *golangsdk.ServiceClient, queueName string) (r GetResult) {
```
Comment (ShiChangkuo) [inline openstack/dli/v1/queues/requests.go:88]: seems the query parameter **queue_type** is missing?

File: openstack/dli/v1/queues/results.go [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+	// Name of a newly created resource queue. The name can contain only digits, letters, and underscores (_), but cannot contain only digits or start with an underscore (_). Length range: 1 to 128 characters.
+	QueueName string `json:"queueName" required:"true"`
+
+	// Description of a queue.
+	Description string `json:"description"`
+
+	Owner string `json:"owner"`
+
+	Create_time int64 `json:"create_time"`
```
Comment (ShiChangkuo) [inline openstack/dli/v1/queues/results.go:21]: **_** is not recommend, please define the field as `CreateTime`

File: openstack/dli/v1/queues/results.go [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+	QueueType string `json:"queueType"`
+
+	// Minimum number of CUs that are bound to a queue. Currently, the value can only be 16, 64, or 256.
+	CuCount int `json:"cuCount" required:"true"`
+
+	// Billing mode of a queue. This value can only be set to 1, indicating that the billing is based on the CUH used.
+	ChargingMode int `json:"chargingMode"`
+
+	//
+	Resource_id string `json:"resource_id"`
```
Comment (ShiChangkuo) [inline openstack/dli/v1/queues/results.go:38]: ditto

File: openstack/dli/v1/queues/requests.go (hunk 3 above)
Comment (hmbig2) [reply openstack/dli/v1/queues/requests.go:88]: tested in console, the queue_name is primary ID , so ignore other parameters

**Pull request merged**